Xamarin.Forms.Core/Internals/Legacy/FileAccess.cs

```diff
@@ -0,0 +1,12 @@
+using System.ComponentModel;
+
+namespace Xamarin.Forms.Internals
+{
+	[EditorBrowsable(EditorBrowsableState.Never)]
+	public enum FileAccess
```

Comment:
are those enums used only for netstandard1.0 ? or everywhere ?

Reply:
only netstandard1.0

Reply:
so wrap them in `#if/#endif` please

Reply:
that's kinda of done my the csproj in a more clean way imo